spec/salestation/web/responses/success_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe Salestation::Web::Responses::Success do
+  subject(:create_success) { described_class.new(attributes) }
+  let(:attributes) { all_attributes }
```

Comment:
I thought about it for other specs, but couldn't come up with anything particularly good.
But for this test specifically you are not removing any attributes.